handler/resource.go

```diff
@@ -13,7 +13,7 @@ import (
 	mysql_r "github.com/tomatool/tomato/resource/database/sql/mysql"
 	postgres_r "github.com/tomatool/tomato/resource/database/sql/postgres"
 	httpclient_r "github.com/tomatool/tomato/resource/http/client"
-	httpserver_r "github.com/tomatool/tomato/resource/http/server"
+	httpwiremock_r "github.com/tomatool/tomato/resource/http/server/wiremock"
```

Comment:
same with this, i think `wiremock_r` should be enough, unless there's `smtpwiremock` or something, wdyt?